packages/default-theme/components/SwFooter.vue

```diff
@@ -1,40 +1,44 @@
 <template>
   <div class="sw-footer">
     <slot class="sw-footer__content" name="content" v-bind="column">
-        <div class="content sw-footer__signature">
-          Made with ❤️ by shopware AG & Vue Storefront
-        </div>
+      <div class="content sw-footer__signature">
+        <SwPluginSlot name="footer-content-before" />
+
+        Made with ❤️ by shopware AG & Vue Storefront
```

Comment:
It is more of a general thing - but I'd like to put it up for discussion. How about if these kinds of elements were itself put into plugin slots.

Few developers will just extend the footer, but instead want to replace the entire content of it. So for now they would have to replace the entire SwFooter component just to get rid of the content.

Maybe something like:

```suggestion
        <SwPluginSlot name="footer-content" default="Made with ❤️ by shopware AG & Vue Storefront" />
```

I am not sure, this lets us easily override or remove the content, but when it comes to deeper nesting this method doesn't work either. But do you get my concern?

Reply:
yes, that's a good point! we can do something like this:
``W
<SwPluginSlot name="footer-content">
  Made with ❤️ by shopware AG & Vue Storefront
</SwPluginSlot>
```

then the plugin may get rid of it by not putting `<slot />` inside its body or place it somewhere. Is that okay? :)

Reply:
https://github.com/DivanteLtd/shopware-pwa/pull/695/commits/b18fb19978927564ad15854dab22dbc2dd3e5303#diff-cfb20b6a177387e46c01724b35356cf7 great input Dominic! Works like a charm and allows both override and extend it :)

Reply:
Great job!